modules/lambda_teams_alert_monitor_log/README.md

```diff
@@ -36,3 +36,4 @@ module "monitor_log" {
 }
 
```

Comment:
Your documentation of the fields is not clear. It's not clear which fields are required and what they mean. I would rather have them in a bullet list or something. For example, where does it say what `entity_name` is supposed to mean? `builds_bucket`?

Reply:
There are also a bunch of variables that have not been documented. Maybe just add a link to `variables.tf` for the documentation?

Reply:
Yes, you are right 💯